stores/hostdb_test.go

```diff
@@ -158,40 +158,98 @@ func (s *SQLStore) addTestScan(hk types.PublicKey, t time.Time, err error, setti
 	})
 }
 
-// TestSQLHosts tests the Hosts method of the SQLHostDB type.
-func TestSQLHosts(t *testing.T) {
```

Comment:
Just because we deprecate the route doesn't mean we need to drop any testing we have for it right now. We still don't want to break it until we actually remove it.